src/installation.md

```diff
@@ -18,7 +18,7 @@ Once that's is all finished, you are ready to [get started with the DM language!
 
 ## VSCode (Very Highly Recommended)
 
-An optional installation to make your life much easier when coding in DM would be VSCode. It's an open-source multi-platform code editor that includes a lot of features and QOL improvements that make it substantially better than the editor that comes with your BYOND download.
+An installation to make your life much easier when coding in DM would be VSCode. It's an open-source multi-platform code editor that includes a lot of features and QOL improvements that make it substantially better than the editor that comes with your BYOND download.
```

Comment:
we must lie to them for their own good 🙏

Reply:
Just remembered from messing with the coding wiki the other day....I wonder if anyone uses Visual Studio instead of VSCode for this... :bigthink: